docs/reference/query.rst

```diff
@@ -163,18 +163,24 @@ and day are optional. If you leave out the day, for example, you will get
 matches for the whole month.
 
 You can also use relative dates to the current time.
-Relative dates begin by the ``@`` character, followed by an optional ``+`` or
-``-`` sign that will increment or decrement now, followed by the time quantity
-that will be represented as an integer followed by either ``d`` for days, ``w``
-for weeks, ``m`` for months and finally ``y`` for year.
+A relative date begins by ``@`` and has to follow the pattern_dq format
+``@([+|-]?)(\d+)([y|m|w|d])``
+- ``@`` indicates it's a date relative to now()
+- the optional ``+`` or ``-`` sign, which defaults to ``+`` will increment or
+decrement now() by a certain quantity
+- that quantity can be expressed in days, weeks, months or years respectively
+'d', 'w', 'm', 'y'
+
+Please note that this relative calculation is rather approximate as it makes
+the assumption of 30 days per month and 365 days per year.
```

Comment:
"Please note that the relative calculation always equates a month to 30 days and a year to 365 days, so it may sometimes be off the mark."

Or something like that. Idk. Your text still seems a bit too stiff and technical. Mine can certainly be improved upon as well. :)

(Feel free to disregard this comment. This tangentiates bikeshedding.)